purchase_ext_sst/views/purchase_order_views.xml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
 
+    <record id="purchase.purchase_rfq" model="ir.actions.act_window">
+        <field name="context">{'search_default_draft':1, 'show_purchase': False}</field>
```

Comment:
Not sure whether we should set `show_purchase` to True to avoid confusion in the form. Let's go as you suggest for now if you have no opinion on this.